### PR TITLE
Update release version to 9.3.0

### PR DESCRIPTION
**Pull request by AlmostMatt**
### Description
Updates version number for 9.3.0 release
***
### Testing
No functional change
***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [x] Other, such as a build process or documentation change.
***

#### Notes
- Bug fixes and feature changes require an update to the `Release Notes` section of `release_build_files/readme.md`.
- Read the contribution guidelines [CONTRIBUTING.md](https://github.com/firebase/firebase-cpp-sdk/blob/main/CONTRIBUTING.md).
- Changes to the public API require an internal API review. If you'd like to help us make Firebase APIs better, please propose your change in a feature request so that we can discuss it together.

### CODE REVIEW

File: release_build_files/readme.md [hunk 1]
```diff
@@ -634,7 +634,7 @@ workflow use only during the development of your app, not for publicly shipping
 code.
 
 ## Release Notes
-### Upcoming Changes
+### 9.3.0
 -   Changes
     - General (Android,Linux): Fixed a concurrency bug where waiting for an
       event with a timeout could occasionally return prematurely, as if the
```
Comment (jonsimantov) [inline release_build_files/readme.md:640]: I know this isn't quite this PR, but is the indentation wrong on the following 2 lines?

Comment (AlmostMatt) [reply release_build_files/readme.md:640]: I think it is correct. All of the later versions also have similar indentation
### 9.0.0
-   Changes
    -   General (iOS): Firebase C++ on iOS is now built using Xcode 13.3.1.

3 spaces between - and Changes, 4 spaces before subsequent dashes

Comment (jonsimantov) [reply release_build_files/readme.md:640]: Are the lines right below this indented the same? (it looks like they aren't in the GitHub UI)

(the line that reads "timeout had occurred" and the following line, I mean)

Comment (AlmostMatt) [reply release_build_files/readme.md:640]: It still looks consistent to me. 6 spaces of indentation for each line that does not have a hyphen.

You can see the raw value of those lines here if you want:
https://raw.githubusercontent.com/firebase/firebase-cpp-sdk/main/release_build_files/readme.md

**Review by DellaBitta — APPROVED** [review]

**Pull request merged**